src/main/java/bio/terra/service/snapshotbuilder/SnapshotAccessRequestModel.java

```diff
@@ -28,7 +28,9 @@ public record SnapshotAccessRequestModel(
     Instant statusUpdatedDate,
     SnapshotAccessRequestStatus status,
     UUID createdSnapshotId,
-    String flightid) {
+    String flightid,
+    String samGroupName,
+    String samGroupCreatedByEmail) {
```

Comment:
I think this actually a Terra ID, which happens to be an email. Not sure the TDR source makes this distinction anywhere though.